### PR TITLE
[FIX] Some typos and adding include modules.

### PR DESCRIPTION
**Pull request by rrahn**

### CODE REVIEW

**Comment by smehringer** [comment]
@eseiler if you don't find something, I think we can merge this.

File: manual/source/Tutorial/HowTo/UseCases/SimpleGenomeAlignment.rst [hunk 1]
```diff
@@ -248,7 +263,7 @@ Final alignment
 
 In the original algorithm, the steps from above would be repeated for the gaps between the anchors selected by the global chaining algorithm.
 In this tutorial we will skip the iterative step and directly compute the final alignment along the global map produced by the chaining algorithm.
-SeqAn offers already an alignment function for filling the gaps and connecting them with the anchors.
+SeqAn offers already an alignment function for filling the gaps and connecting them with the anchors, which is available in the ``seeds`` module.
```
Comment (eseiler) [inline manual/source/Tutorial/HowTo/UseCases/SimpleGenomeAlignment.rst]: SeqAn already offers 💅

Comment (smehringer) [reply manual/source/Tutorial/HowTo/UseCases/SimpleGenomeAlignment.rst]: well now we have to wait for @rrahn :laughing:

**Comment by smehringer** [comment]
@eseiler Could you just replicate this PR with the adjusted changes? That we can close this PR and merge yours.

**Pull request closed without merging**